sensor_msgs/Image image
sensor_msgs/Image mask
---
int32 testNumber
std_msgs/String label
